integration/e2e/config-all-in-one-gcs.yaml

```diff
@@ -38,9 +38,11 @@ storage:
       bucket_name: tempo
       endpoint: https://tempo_e2e-gcs:4443/storage/v1/
       insecure: true
+      obj_attrs:
+        cachecontrol: "no-cache"
```

Comment:
does this work? will it set this field:

https://github.com/googleapis/google-cloud-go/blob/main/storage/storage.go#L1263

I wonder if we can only expose the ability to configure this:

https://github.com/googleapis/google-cloud-go/blob/main/storage/storage.go#L1326

and then if the user specifies a key of `Cache-Control` it will be magically translated into the Fixed-key metadata:

https://cloud.google.com/storage/docs/metadata